src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -90,9 +89,7 @@ def _validate_cidr(rule):
         except (ipaddress.NetmaskValueError, ValueError) as error:
             raise SpinnakerSecurityGroupCreationFailed(error)
 
-        if network.prefixlen < 13:
-            msg = 'The network range ({0}) specified is too open.'.format(rule['app'])
-            raise SpinnakerSecurityGroupCreationFailed(msg)
```

Comment:
Should we log warning instead of removing this if statement?

Reply:
I wouldn't want any issues in the OSS release as companies may have different requirements. Good fix.